spec/index.html

```diff
@@ -1366,7 +1366,7 @@ <h2>Motivation</h2>
           pending feedback from the community and new knowledge gained
           from practical experience of using the basic version
           of YAML-LD that we will henceforth call
-          <strong>JSON Profile of YAML-LD</strong>.
+          <strong>Basic Profile of YAML-LD</strong>.
```

Comment:
This is the only instance of `Basic Profile of YAML-LD`; all others are `YAML-LD Basic Profile`, so I think should be changed to match the others.
```suggestion
          <strong>YAML-LD Basic Profile</strong>.
```